package.json

```diff
@@ -18,18 +18,26 @@
     "internal"
   ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint": "eslint --env node --ignore-pattern tmp/ --config test/core-config.json .",
+    "test": "eslint --config test/core-config.json --ignore-path tmp/.eslintignore --no-eslintrc tmp/"
+  },
+  "engines": {
+    "node": "12.x || 14.x || >= 16"
   },
   "peerDependencies": {
-    "eslint": "^7.0.0"
+    "eslint": ">=6"
   },
   "dependencies": {
     "@babel/core": "^7.16.7",
     "@babel/eslint-parser": "^7.16.5",
     "@babel/plugin-syntax-class-properties": "^7.12.13",
     "@babel/plugin-syntax-import-assertions": "^7.16.7",
     "@babel/plugin-syntax-top-level-await": "^7.14.5",
-    "eslint-plugin-markdown": "^2.2.1",
-    "eslint-plugin-node-core": "^1.0.0"
+    "eslint-plugin-jsdoc": "^37.6.1",
+    "eslint-plugin-markdown": "^2.2.1"
+  },
+  "devDependencies": {
+    "eslint": "^8.6.0",
```

Comment:
I think the version here should match the lowest in `peerDependencies`, otherwise we could use features/rules that don't exist in ESLint 6 and 7.

Reply:
Core is already using 8

Reply:
But it would probably be good to test a mix later on

Reply:
If 8 is needed, then let's update the peerDependencies field.

Reply:
yeah, it's what I had originally, but I used the range to match what the eslint plugin yeoman generator uses